src/selectors/blockchain.ts

```diff
@@ -22,6 +22,8 @@ export const findDefaultProvider = (state: State) => {
   })
 }
 
+export const getActiveProvider = (state: State): Providers[string] => selector(state).activeProvider
```

Comment:
`Providers[string]` is `any`
To grab from Providers values you need `Providers[keyof Providers]`
Another way would be to specify possible providers:
```
type ProviderName = 'METAMASK' | 'MIST'
interface Provider {
 [P in ProviderName]: any
}
```
then `(state: State): ProviderName`
also we'll need Provider interface at some point

Reply:
achso ok ill take a look and edit these, cheers